keycloak/templates/keycloak.conf.j2

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=Keycloak
+After=network.target
+
+[Service]
+Type=idle
+User=keycloak
+ExecStart=/opt/keycloak/bin/kc.sh start-dev
+Environment="KC_DB={{ kc_db }}"
+Environment="KC_DB_URL={{ kc_db_url }}"
+Environment="KC_DB_USERNAME={{ kc_db_username }}"
+Environment="KC_DB_PASSWORD={{ kc_db_password }}"
```

Comment:
We should probably escape special characters here since they often appear in passwords. (you can look in the `service` role to see how to escape characters in SystemD config files.

Reply:
done